server/api/models/user.ts

```diff
@@ -18,6 +18,10 @@ let flow = Schema({
     type: String,
     required: true,
   },
+  meta: {
```

Comment:
Need to understand the relevance of `meta` here. Why is storing this data necessary?